app/assets/stylesheets/config/_base.css

```diff
@@ -0,0 +1,48 @@
+:root {
+  --script-size-s: 3.25rem;
+  --script-size-m: 4.5rem;
+  --script-size-l: 5.5rem;
+  --font-size-xs: 1.6rem;
+  --font-size-s: 2rem;
+  --font-size-m: 2.5rem;
+  --font-size-l: 4rem;
+  --font-size-xl: 6.5rem;
+}
+
+*, *::before, *::after {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
```

Comment:
**sugestion:** It's a matter of personal preference, but I like to omit units from `0` values

```suggestion
  margin: 0;
  padding: 0;
```